### PR TITLE
Redirect to dftd3 and dftd4 package

### LINKED ISSUE #3: Interface to dftd4
See relevant discussions in
https://github.com/pyscf/pyscf/issues/983
https://github.com/dftd4/dftd4/issues/107

### PR DESCRIPTION
**Pull request by sunqm**
Fix #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
+pip install dftd3 dftd4
 ```
 
-* Install in a custom folder for development
+dftd3 package provides a drop-in replacement of pyscf.dftd3.itrf. For example
+
 ```
-git clone https://github.com/pyscf/dftd3 /home/abc/local/path
+from pyscf import gto
+import dftd3.pyscf as itrf
```
Comment (awvwgk) [inline README.md]:
```suggestion
import dftd3.pyscf as d3
```

File: README.md [hunk 2]
```diff
@@ -1,24 +1,35 @@
...
+
+mol = gto.M(
+    atom = ''' O    0.00000000    0.00000000   -0.11081188
+               H   -0.00000000   -0.84695236    0.59109389
+               H   -0.00000000    0.89830571    0.52404783 ''',
+    basis = 'cc-pvdz')
+
+mf = itrf.dftd3(mol.RHF())
```
Comment (awvwgk) [inline README.md]:
```suggestion
mf = d3.energy(mol.RHF())
```

File: README.md [hunk 3]
```diff
@@ -1,24 +1,35 @@
...
-echo 'export PYSCF_EXT_PATH=/home/abc/local/path:$PYSCF_EXT_PATH' >> ~/.bashrc
+mf.Gradients()
+mf.kernel()
 ```
 
-You can find more details of extended modules in the document
-[extension modules](http://pyscf.org/install.html#extension-modules)
+See also discussions in https://github.com/pyscf/dftd3/issues/3
+and the instructions for dftd3
+https://github.com/awvwgk/simple-dftd3/blob/main/python/README.rst#pyscf-support
```
Comment (awvwgk) [inline README.md]:
```suggestion
https://dftd3.readthedocs.io/en/latest/api/pyscf.html
```

File: README.md [hunk 4]
```diff
@@ -1,24 +1,35 @@
...
+and dftd4
+https://github.com/dftd4/dftd4/tree/main/python#pyscf-support
```
Comment (awvwgk) [inline README.md]:
```suggestion
https://dftd4.readthedocs.io/en/latest/reference/pyscf.html
```

File: examples/00-hf_with_dftd3.py [hunk 5]
```diff
@@ -9,7 +9,7 @@
 
 from pyscf import gto
 from pyscf import scf
-from pyscf import dftd3
+import dftd3.pyscf as itrf
```
Comment (awvwgk) [inline examples/00-hf_with_dftd3.py]:
```suggestion
import dftd3.pyscf as d3
```

File: examples/00-hf_with_dftd3.py [hunk 6]
```diff
@@ -18,6 +18,8 @@
 mol.basis = 'cc-pvdz'
 mol.build()
 
-mf = dftd3.dftd3(scf.RHF(mol))
+mf = itrf.dftd3(scf.RHF(mol))
```
Comment (awvwgk) [inline examples/00-hf_with_dftd3.py]:
```suggestion
mf = d3.energy(scf.RHF(mol))
```

**Review by awvwgk — CHANGES_REQUESTED** [review]

**Comment by sunqm** [comment]
@awvwgk thanks for the suggestions. Let us wait for a few days to see if there are more comments. Then I'll update this repo and the pyscf-dftd3 package on pypi

**Pull request merged**